src/components/Conversion/ConversionNOB.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <section id="nature-of-business-section">
+    <v-row no-gutters>
+      <v-col cols="12" sm="3" class="pr-4">
+        <label>Nature of Business</label>
+        <v-chip
+          v-if="hasNatureOfBusinessChanged"
+          id="changed-chip"
+          x-small label
+          color="primary"
+          text-color="white"
+        >
+          Changed
+        </v-chip>
+      </v-col>
+
+      <v-col cols="12" sm="9">
+        <div v-if="onEditMode">
+          <p class="ma-0">
+            Provide a brief description of the nature of business (e.g., corner grocery store,
+            automotive repair service, landscaping, etc.).
+          </p>
+          <v-text-field
```

Comment:
Is this supposed to be multi-line? (I haven't seen the designs, sorry.)

If so, consider using v-textarea instead.